dialogs.py

```diff
@@ -102,80 +107,82 @@ def on_button(value):
     dialog.minsize(320, dialog_height)
 
     dialog.transient(parent)
+    
     dialog.grab_set()
-
     dialog.wait_window()
     return result
 
 
-def show_message(title="Title", details="Description", *, parent=None, icon=None):
+def show_message(title="Title", details="Description", *, parent=None, icon=None, theme="light"):
     return popup(
         parent,
         title,
         details,
         icon,
         buttons=[("Ok", None, "default")],
+        theme=theme,
     )
 
 
-def ask_ok_cancel(title="Title", details="Description", *, parent=None, icon=None):
+def ask_ok_cancel(title="Title", details="Description", *, parent=None, icon=None, theme="light"):
     return popup(
         parent,
         title,
         details,
         icon,
         buttons=[("Ok", True, "accent"), ("Cancel", None)],
+        theme=theme,
     )
 
 
-def ask_yes_no(title="Title", details="Description", *, parent=None, icon=None):
+def ask_yes_no(title="Title", details="Description", *, parent=None, icon=None, theme="light"):
     return popup(
         parent,
         title,
         details,
         icon,
         buttons=[("Yes", True, "accent"), ("No", False)],
+        theme=theme,
     )
 
 
-def ask_yes_no_cancel(title="Title", details="Description", *, parent=None, icon=None):
+def ask_yes_no_cancel(title="Title", details="Description", *, parent=None, icon=None, theme="light"):
     return popup(
         parent,
         title,
         details,
         icon,
         buttons=[("Yes", True, "accent"), ("No", False), ("Cancel", None)],
+        theme=theme,
     )
 
 
-def ask_retry_cancel(title="Title", details="Description", *, parent=None, icon=None):
+def ask_retry_cancel(title="Title", details="Description", *, parent=None, icon=None, theme="light"):
     return popup(
         parent,
         title,
         details,
         icon,
         buttons=[("Retry", True, "accent"), ("Cancel", None)],
+        theme=theme,
     )
 
 
-def ask_allow_block(title="Title", details="Description", *, parent=None, icon=None):
+def ask_allow_block(title="Title", details="Description", *, parent=None, icon=None, theme="light"):
     return popup(
         parent,
         title,
         details,
         icon,
         buttons=[("Allow", True, "accent"), ("Block", False)],
+        theme=theme,
     )
 
 
 if __name__ == "__main__":
     window = tk.Tk()
-
-    window.tk.call("source", "sun-valley.tcl")
-    window.tk.call("set_theme", "dark")
-
     window.geometry("600x600")
 
-    show_message("No WiFi connection", "Check your connection and try again.")
+    ask_allow_block("No WiFi connection", "Check your connection and try again.", theme = "dark")
```

Comment:
```suggestion
show_message("No WiFi connection", "Check your connection and try again.")
```

This is totally unrelated.

Reply:
Sorry, I was testing the messagebox and I forgot to change it back.